### DISCUSSION TITLE
sbas 提示维度不匹配

### DISCUSSION THREAD

**Discussion by xinchao131** [gmtsar/gmtsar#1011] [Q&A] [answered]
输入命令sbas intf.tab scene.tab 9 5 812 1000 -atm 2 -smooth 1.0 -wavelength 0.0554658 -incidence 38 -range 875267.97 -rms -dem
gmtsarVB:~/F2/SBAS> sbas intf.tab scene.tab 9 5 812 1000 -atm 2 -smooth 1.0 -wavelength 0.0554658 -incidence 38 -range 875267.97 -rms -dem

compute RMS misfit
compute DEM error

setting smoothing to   1.000 
setting radar wavelength to   0.055 m 
setting radar incidence angle to  38.000 degree 
setting range to 875267.970 m 
Required Memory Usage is 0.121011 GB ...
Memory Allocation Successful...
read table file ...
number of SAR scenes is 5 
number of interferograms is 9 
read phase and correlation grids ...
 dimension don't match! ../intf_all/2022304_2022316/corr.grd 
就报错了，求大佬解惑

**Comment by xinchao131** [comment]
已解决

**Reply by Xiaohua-Eric-Xu** [reply, marked as answer]
LOL, answering for others who may have similar issue. To check the dimension. run 
```
gmt grdinfo ../intf_all/2022304_2022316/corr.grd
```
and check whether the dimensions match the input dimension you input for sbas.

**Reply by xinchao131** [reply]
thank you sir
